commands/cloudapi-v5/backupunit.go

```diff
@@ -182,6 +183,7 @@ Required values to run command:
 		return completer.BackupUnitsIds(os.Stderr), cobra.ShellCompDirectiveNoFileComp
 	})
 	deleteCmd.AddBoolFlag(config.ArgWaitForRequest, config.ArgWaitForRequestShort, config.DefaultWait, "Wait for the Request for BackupUnit deletion to be executed")
+	deleteCmd.AddBoolFlag(cloudapiv5.ArgAll, cloudapiv5.ArgAllShort, false, "delete all BackupUnits.")
```

Comment:
Please use upper case at the beginning of flag's description. It is good practice to be in sync with the rest of the flags. This applies for ALL the places in this PR where the flag is added.